airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -200,6 +200,65 @@
                 "order": 3
               }
             }
+          },
+          {
+            "title": "Azure Blob Staging",
+            "additionalProperties": false,
+            "description": "Writes large batches of records to a file, uploads the file to Azure Blob Storage, then uses <pre>COPY INTO table</pre> to upload the file. Recommended for large production workloads for better speed and scalability.",
+            "required": [
+              "method",
+              "azure_blob_storage_account_name",
+              "azure_blob_storage_container_name",
+              "azure_blob_storage_account_key",
+              "snowflake_azure_external_stage_name"
+            ],
+            "properties": {
+              "method": {
+                "type": "string",
+                "enum": ["Blob Staging"],
```

Comment:
I would recommend changing this to `Azure Blob Staging` to make it clear what blob is referring to